back/spec/policies/idea_policy_spec.rb

```diff
@@ -195,6 +195,85 @@
     end
   end
 
+  context 'on a published native survey response in a public project' do
+    let!(:proposed_status) { create(:idea_status_proposed) }
+    let(:idea) { create(:native_survey_response) }
+    let(:project) { idea.project }
+
+    context 'for a visitor' do
+      let(:user) { nil }
+
+      it do
+        is_expected.not_to permit(:show)
+        is_expected.not_to permit(:by_slug)
+        is_expected.not_to permit(:update)
+        is_expected.not_to permit(:destroy)
+        is_expected.not_to permit(:index_xlsx)
+
+        expect(scope.resolve.size).to eq 0
+      end
+    end
+
+    context 'for a resident who is not the response author' do
+      let(:user) { create(:user) }
+
+      it do
+        is_expected.not_to     permit(:show)
+        is_expected.not_to     permit(:by_slug)
+        is_expected.not_to permit(:create)
+        is_expected.not_to permit(:update)
+        is_expected.not_to permit(:destroy)
+        is_expected.not_to permit(:index_xlsx)
+
+        expect(scope.resolve.size).to eq 0
+      end
+    end
+
+    context 'for a resident who is the response author' do
+      let(:user) { idea.author }
+
+      it do
+        is_expected.to permit(:show)
+        is_expected.to permit(:by_slug)
+        is_expected.to permit(:create)
+        is_expected.not_to permit(:update)
+        is_expected.not_to permit(:destroy)
+        is_expected.not_to permit(:index_xlsx)
+
+        expect(scope.resolve.size).to eq 1
+      end
+    end
+
+    context 'for an admin' do
+      let(:user) { create(:admin) }
+
+      it do
+        is_expected.to permit(:show)
+        is_expected.to permit(:by_slug)
```

Comment:
This is a change from how it's implemented currently, because I also had to chance some acceptance specs to checked this to be unauthorized.
I decided to change it, because I don't see why this should be the case. Admins can export all survey responses, and can see them through Sensemaking, so the policy should reflect that.